bundlers/bundler/src/types.ts

```diff
@@ -118,3 +118,501 @@ export type SolidityResolver = (
 	 */
 	resolveEsmModuleSync: (module: string, basedir: string) => BundlerResult
 }
+
+export interface CompilerOptions<Filename extends string> {
```

Comment:
Where did you get this from? Did you write this yourself?

Reply:
Compiler options: https://docs.soliditylang.org/en/latest/using-the-compiler.html#input-description

Compiler output: https://docs.soliditylang.org/en/latest/metadata.html#contract-metadata